types/javascript-color-gradient/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for javascript-color-gradient 1.3
+// Type definitions for javascript-color-gradient 2.3
```

Comment:
The latest version seems to be 2.4 according to npm

```suggestion
// Type definitions for javascript-color-gradient 2.4
```

Reply:
My bad, good catch!